lib/ae_mdw/blocks.ex

```diff
@@ -259,10 +259,13 @@ defmodule AeMdw.Blocks do
 
     header = :aec_db.get_header(mb_hash)
 
+    gas = mb_hash |> :aec_db.get_block() |> :aec_blocks.gas()
```

Comment:
Can we instead fetch the block and then get the header from it? That way we avoid fetching it twice
```elixir
block = :aec_db.get_block(mb_hash)
header = :aec_blocks.to_header(block)
gas = :aec_blocks.gas(block)
```